.github/workflows/checks.yml

```
@@ -0,0 +1,28 @@
+        - opened
+        - synchronize
+        - reopened
+    branches:
+      - dev
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)